web/src/components/ChatMessage.tsx

```diff
@@ -57,9 +57,12 @@ const getIcon = (userIsAuthor: boolean, isAutomaticAnswer: boolean, t: TFunction
   return <Icon src={icon} title={isAutomaticAnswer ? t('bot') : t('human')} />
 }
 
-const ChatMessage = ({ message, showIcon }: ChatMessageProps): ReactElement => {
+const ChatMessage = ({ message, previousMessage }: ChatMessageProps): ReactElement => {
   const { t } = useTranslation('chat')
   const { body, userIsAuthor, isAutomaticAnswer } = message
+  const hasAuthorChanged = message.userIsAuthor !== previousMessage?.userIsAuthor
+  const hasAutomaticAnswerChanged = message.isAutomaticAnswer !== previousMessage?.isAutomaticAnswer
+  const showIcon = previousMessage ? hasAuthorChanged || hasAutomaticAnswerChanged : true
```

Comment:
:wrench: This can be simplified, if previous message is undefined both hasAuthorChanged and hasAutomaticAnswerChanged are true already:
```suggestion
  const showIcon = hasAuthorChanged || hasAutomaticAnswerChanged
```

Reply:
Very nice catch